src/pages/home/home.html

```diff
@@ -66,8 +66,8 @@ <h2 margin-bottom>{{'headings.myActiveAssessment' | translate}}</h2>
     </ion-card>
   </div> -->
 
-  <div>
-    <h2 margin>{{'headings.myActiveAssessment' | translate}}</h2>
+  <div margin>
+    <h2>{{'headings.myActiveAssessment' | translate}}</h2>
     <!-- 
```

Comment:
when changing margin for graph this margin on h2 was not working so added in div